src/main/java/org/takes/facets/fallback/FbStatus.java

```diff
@@ -41,11 +42,17 @@
  *
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
- * @version $Id$
+ * @version $Id: 0f621de76a6b6bcc446242289b2e8ae1b4a67fc4 $
```

Comment:
@dalifreire remove the unneeded string at the end, this field should be `@version $Id$` only